pkg/iso9660util/iso9660util.go

```diff
@@ -60,3 +60,18 @@ func WriteFile(fs filesystem.FileSystem, path string, r io.Reader) (int64, error
 	defer f.Close()
 	return io.Copy(f, r)
 }
+
+func IsISO9660(imagePath string) (bool, error) {
```

Comment:
Fast path: `if strings.HasSuffix(strings.ToLower(imagePath), ".iso") { return true, nil }`

Reply:
Sounds good, I switched to that one. Also cannot fail. 😄

Reply:
I had to revert that change because I call it on "baseDisk", which has no file extension.

Anyways, even the "proper" implementation is pretty fast, just opening a file and reading a 32KB chunk; this should not be noticeable compare to everything else that is happening.